vue/components/ui/molecules/carousel/carousel.vue

```diff
@@ -0,0 +1,283 @@
+<template>
+    <div
+        class="carousel"
+        v-bind:style="style"
+        ref="carousel"
+        v-on:mousedown="onMouseDown"
+        v-on:mouseup="onMouseUp"
+        v-on:mouseleave="onMouseLeave"
+        v-on:touchstart="onTouchStart"
+        v-on:touchend="onTouchEnd"
+        v-on:touchcancel="onTouchCancel"
+        v-on:touchleave="onTouchLeave"
+    >
+        <icon
+            class="arrow arrow-previous"
+            v-bind:color="arrowsColor"
+            v-bind:icon="'chevron-left'"
+            v-bind:width="36"
+            v-bind:height="36"
+            v-show="arrowsVisibility"
+            v-on:click="previous"
+        />
+        <transition-group class="slide-container" tag="div" name="fade">
+            <div
+                class="slide"
+                v-for="(image, index) in images"

```

Comment:
Do something similar to [radio-group](https://github.com/ripe-tech/ripe-components-vue/blob/master/vue/components/ui/molecules/radio-group/radio-group.vue) where each `item` is an object. This way we have the ground to add more info in the future, as opposed to simply having a string.

Reply:
Notice that a carousel is similar to a radio group in the sense that you are picking 1 value among several.